applications/nrf_desktop/configuration/nrf54h20dk_nrf54h20_cpuapp/app_common.dtsi

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+/ {
+	/* Redefine leds to fit CAF requirements. */
+	/delete-node/ leds;
+
+	leds0 {
+		compatible = "gpio-leds";
+		status = "okay";
+		label = "System State LED";
+
+		led0: led_0 {
+			gpios = < &gpio9 0 GPIO_ACTIVE_HIGH >;
+			label = "Green LED 0";
+		};
+	};
+
+	leds1 {
+		compatible = "gpio-leds";
+		status = "okay";
+		label = "Peer State LED";
+
+		led1: led_1 {
+			gpios = < &gpio9 1 GPIO_ACTIVE_HIGH >;
+			label = "Green LED 1";
+		};
+	};
+
+	leds2 {
+		compatible = "gpio-leds";
+		status = "disabled";
+		label = "Unused LED 2";
+
+		led2: led_2 {
+			gpios = < &gpio9 2 GPIO_ACTIVE_HIGH >;
+			label = "Green LED 2";
+		};
+	};
+
+	leds3 {
+		compatible = "gpio-leds";
+		status = "disabled";
+		label = "Unused LED 3";
+
+		led3: led_3 {
+			gpios = < &gpio9 3 GPIO_ACTIVE_HIGH >;
+			label = "Green LED 3";
+		};
+	};
+
+	hid_dev_0: hid_dev_0 {
```

Comment:
Do we want to define USB HID instances in common file or rather per build type?

Reply:
Improved in the following PR:

https://github.com/nrfconnect/sdk-nrf/pull/19239